client/components/homepage/Customers.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Customers = () => (
+  <section id="customers" className="section background-gray-lightest padding--small">
+    <div className="container">
+      <div className="row">
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/hackreactor.png" title="Hack Reactor" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/telegraph-academy.png" title="Telegraph Academy" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/google.png" title="Google" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/amazon.png" title="Amazon" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/hackreactor.png" title="Hack Reactor" alt="" className="img-responsive" /></div>
+        </div>
+        <div className="col-sm-4 col-md-2 col-xs-6">
+          <div className="customer"><img src="../img/hackreactor.png" title="Hack Reactor" alt="" className="img-responsive" /></div>
+        </div>
```

Comment:
Looks like each of these customer sections are pretty similar as well. Probably worth creating a reusable component here as well.